our_site/planner/views.py

```diff
@@ -1,33 +1,69 @@
 from django.http import HttpResponse, HttpResponseRedirect
 from django.template import loader
+from .models import Event
 import django.forms as forms
+from django.shortcuts import render
 
 
-class DashboardFilterAllEvents(forms.Form):
-    pass
 
+def dashboard(request): 
+    owner = None
+    event_list = []
 
-class DashboardFilterMyEvents(forms.Form):
-    pass
+    class DashboardFilterAllEvents(forms.Form):
+        pass
 
 
-class DashboardFilterInvitedEvents(forms.Form):
-    pass
+    class DashboardFilterMyEvents(forms.Form):
+        pass
+       
 
+    class DashboardFilterInvitedEvents(forms.Form):
+        pass
 
-def dashboard(request):
-    if request.user.is_anonymous:  # User is redirected to log in if they are not logged in
+
+    if request.user.is_anonymous:
         return HttpResponseRedirect("/account/login/")
 
+    if request.user.is_authenticated:
```

Comment:
Why are we redirecting the user if they are authenticated? Isn't this the view we want them to have?

Reply:
that just declared who the owner was but i just moved that to the top of the code now